src/epfl-gallery/index.js

```diff
@@ -53,7 +53,7 @@ registerBlockType( 'epfl/gallery', {
         return (
             <Fragment>
                 <InspectorControls>
-                    <p><a className="wp-block-help" href={ __('https://www.epfl.ch/campus/services/en/it-services/web-services/wordpress-help/text-tools-en/gallery/', 'epfl') } target="new">{ __('Online help', 'epfl') } </a></p>
+                    <p><a className="wp-block-help" href={ __('https://www.epfl.ch/campus/services/website/en/it-services/web-services/wordpress-help/text-tools-en/gallery/', 'epfl') } target="new">{ __('Online help', 'epfl') } </a></p>
```

Comment:
```suggestion
                    <p><a className="wp-block-help" href={ __('https://www.epfl.ch/campus/services/website/gallery/', 'epfl') } target="new">{ __('Online help', 'epfl') } </a></p>
```
je suggèrerais d'avoir le même format que les autres blocs en faite, avec juste le slug dans l'URL, ce qui permet d'être plus flexible dans l'organisation des pages au sein du site de documentation.